.github/labeler.yml

```
@@ -0,0 +1,5 @@
+"Type Changes":
+  - all:
+    - changed-files:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)